airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/manifest.yaml

```
@@ -0,0 +1,924 @@
+version: 0.65.0
+type: DeclarativeSource
+
+definitions:
+  oauth_authenticator:
+    type: OAuthAuthenticator
+    client_id: "{{ config['credentials']['client_id'] }}"
+    client_secret: "{{ config['credentials']['client_secret'] }}"
+    refresh_token: "{{ config['credentials']['refresh_token'] }}"
+    refresh_request_body: {}
+    token_refresh_endpoint: "{{ 'https://sandbox-lever.auth0.com/oauth/token' if config[ 'environment' ] == 'Sandbox' else 'https://auth.lever.co/oauth/token' }}"
+    grant_type: refresh_token
+
+  basic_authenticator:
+    type: BasicHttpAuthenticator
+    username: "{{ config[''credentials''][''api_key''] }}"
```

Please fix the quotes:
```suggestion
    username: "{{ config['credentials']['api_key'] }}"
```